src/core/lib/security/credentials/tls/grpc_tls_crl_provider.cc

```diff
@@ -57,11 +57,13 @@ std::string IssuerFromCrl(X509_CRL* crl) {
   if (crl == nullptr) {
     return "";
   }
-  char* buf = X509_NAME_oneline(X509_CRL_get_issuer(crl), nullptr, 0);
-  std::string ret;
-  if (buf != nullptr) {
-    ret = buf;
-  }
+  X509_NAME* issuer = X509_CRL_get_issuer(crl);
```

Comment:
Can we also make this code block into a helper function in `ssl_transport_security_utils.{h,cc}`? This way it can be reused elsewhere if needed, and we can test it in the same place as the similar APIs.

Reply:
Thanks for investigating. I think it would be good to pull out the `ssl_transport_security_util` into its own BUILD target as you suggest, so that we can avoid duplicating OpenSSL code, avoid divergent behavior between the copies of the code, etc. Totally fine doing this in a separate PR though. Can you leave a TODO for this?

Reply:
Done